core/storage/user/gae_models.py

```diff
@@ -63,6 +63,13 @@ class UserSettingsModel(base_models.BaseModel):
         choices=[
             constants.DASHBOARD_TYPE_LEARNER,
             constants.DASHBOARD_TYPE_CREATOR])
+    # The preferred dashboard display preference.
+    creator_dashboard_display = ndb.StringProperty(
+        default=feconf.CREATOR_DASHBOARD_DISPLAY_PREF_CARD,
+        indexed=False,
+        choices=[
```

Comment:
Refer to feconf.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS

Reply:
Done